src/Features/Core/Portable/Navigation/NavigationOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.Navigation
+{
+    internal static class NavigationOptions
+    {
+        public const string FeatureName = "Navigation";
+
+        public static readonly Option<bool> UsePreviewTab = new Option<bool>(FeatureName, "UsePreviewTab", defaultValue: false);
```

Comment:
Can you update the docs to just to clarify this doesn't reflect the user option but is just a "try to use it if it's on" flag? And per our :ice_cream: conversation, and I suspect the default should true and the places we're passing false is code we should be deleting...I'm happy to file a bug to track that in the distant future.